src/Microsoft.Net.Http.Headers/SetCookieHeaderValue.cs

```diff
@@ -452,9 +452,16 @@ private static int GetSetCookieLength(StringSegment input, int startIndex, out S
                     result.HttpOnly = true;
                 }
                 // extension-av = <any CHAR except CTLs or ";">
-                else
-                {
-                    // TODO: skip it? Store it in a list?
+                else 
+                {   
+                    // TODO: skiping it for now to avoid parsing failure? Store it in a list?
+                    // = (no spaces)
+                    if (!ReadEqualsSign(input, ref offset))
+                    {
+                        return 0;
+                    }
+                    // We don't do any detailed validation on the domain.
```

Comment:
Can you clean up these comments? They look like they were copied from elsewhere.